modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-enable-node-scan-scannerv4_{context}"]
+= Enabling {op-system} node scanning with Scanner V4
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+
+:FeatureName: RHCOS node scanning with Scanner V4
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* For scanning {op-system} node hosts of the secured cluster, you must have installed the following:
+** Secured Cluster services on {ocp} {ocp-supported-version} or later
+** {product-title-short} version 4.6 or later
```

Comment:
According to the ISG, [_the following_ takes a noun](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording), such as:
```suggestion
* For scanning {op-system} node hosts of the secured cluster, you must have installed the following software:
** Secured Cluster services on {ocp} {ocp-supported-version} or later
** {product-title-short} version 4.6 or later
```